<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileVpzRtool</class>
 <widget class="QWidget" name="fileVpzRtool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="modelTree">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackProperty">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBlank"/>
     <widget class="QWidget" name="tabModelProperty">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="modelProperty">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>3</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Property</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Model name</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>Dynamic</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeft|AlignTop</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Conditions</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeft|AlignTop</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
